galaxy_ng/app/migrations/0026_move_perms_to_roles.py

```diff
@@ -0,0 +1,191 @@
+        ("container", "namespace_modify_content_containerpushrepository"),
+        ("container", "namespace_push_containerdistribution"),
+        ("container", "add_containernamespace"),
+        ("container", "change_containernamespace"),
+    ), "galaxy.execution_environment_admin"),
+    ((
+        ("ansible", "modify_ansible_repo_content"),
+    ), "galaxy.content_admin"),
+    ((
+        ("galaxy", "add_namespace"),
+        ("galaxy", "change_namespace"),
+        ("galaxy", "delete_namespace"),
+        ("galaxy", "view_namespace"),
+        ("galaxy", "upload_to_namespace"),
+        ("ansible", "delete_collection"),
+    ), "galaxy.namespace_owner"),
+    ((
+        ("galaxy", "upload_to_namespace"),
+        ("ansible", "delete_collection"),
+    ), "galaxy.publisher"),
+    ((
+        ("galaxy", "view_group"),
+        ("galaxy", "delete_group"),
+        ("galaxy", "add_group"),
+        ("galaxy", "change_group"),
+    ), "galaxy.group_admin"),
+    ((
+        ("galaxy", "view_user"),
+        ("galaxy", "delete_user"),
+        ("galaxy", "add_user"),
+        ("galaxy", "change_user"),
+    ), "galaxy.user_admin"),
+    ((
+        ("galaxy", "add_synclist"),
+        ("galaxy", "change_synclist"),
+        ("galaxy", "delete_synclist"),
+        ("galaxy", "view_synclist"),
+        ("ansible", "change_collectionremote"),
+    ), "galaxy.synclist_owner"),
+]
+
+GROUP_OBJECT_PERMISSION_TRANSLATOR = [
+    ("galaxy", "upload_to_namespace"),
+]
+
+USER_OBJECT_PERMISSION_TRANSLATOR = [
+    ("container", "view_containernamespace"),
+    ("container", "delete_containernamespace"),
+    ("container", "change_containernamespace"),
+    ("container", "namespace_add_containerdistribution"),
+    ("container", "namespace_delete_containerdistribution"),
+    ("container", "namespace_view_containerdistribution"),
+    ("container", "namespace_pull_containerdistribution"),
+    ("container", "namespace_push_containerdistribution"),
+    ("container", "namespace_change_containerdistribution"),
+    ("container", "namespace_view_containerpushrepository"),
+    ("container", "namespace_modify_content_containerpushrepository"),
+]
+
+
+def remove_locked_perms_from_permissions(perms_to_remove, permissions):
+    # permissions_to_remove: list of Permission objects to remove
+    # permissions: queryset of Permission objects
+    for p in perms_to_remove:
+        permissions = permissions.exclude(id=p.id)
+    return permissions
+
+
+def move_permissions_to_roles(apps, schema_editor):
+    Group = apps.get_model("galaxy", "Group")
+    GroupRole = apps.get_model("core", "GroupRole")
+    UserRole = apps.get_model("core", "UserRole")
+    Permission = apps.get_model("auth", "Permission")
+    Role = apps.get_model("core", "Role")
+    UserObjectPermission = apps.get_model("guardian", "UserObjectPermission")
+    GroupObjectPermission = apps.get_model("guardian", "GroupObjectPermission")
```

Comment:
This reminds me that we need to come up with an exit strategy eventually when we do not even have guardian installed anymore. Maybe we can just assume that there is nothing to do at that point as it must have been a fresh install.